models/role.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+//Roles represents an array of the role model
+type Roles struct {
+	Count      int    `json:"count"`
+	Offset     int    `json:"offset"`
+	Limit      int    `json:"limit"`
+	Items      []Role `json:"items"`
+	TotalCount int    `json:"total_count"`
+}
+
+//Role represents the structure for a role
+type Role struct {
+	ID          string   `bson:"_id"`
```

Comment:
does this need a `json:"id"`? or does it default to `id` when marshalled to json?

Reply:
This is a good point, I have added in the json:id now